crates/trigger/src/cli.rs

```diff
@@ -243,6 +254,16 @@ async fn warn_slow_wasm_build() {
     println!();
 }
 
+// Parse the key/values passed in as `key=value` pairs.
+// TODO: `spin-utils` WHEN
+fn parse_kv(s: &str) -> Result<(String, String)> {
+    let parts: Vec<_> = s.splitn(2, '=').collect();
+    if parts.len() != 2 {
+        bail!("Key/Values must be of the form `key=value`");
+    }
+    Ok((parts[0].to_owned(), parts[1].to_owned()))
```

Comment:
Slightly nicer:
```suggestion
    let Some((key, value)) = s.split_once('=') else {
        bail!("Key/Values must be of the form `key=value`");
    }
    Ok((key.to_owned(), value.to_owned()))
```

Reply:
Not only nicer, but more correct (if a value contains an `=` sign e.g. base64).  But I wanted to keep it exactly the same as the `deploy` one (and didn't want to change `deploy`) so that when spin-utils came along we wouldn't need to eyeball them for subtle differences.

But yeah now I can (rebase and) move to common.

Reply:
I think `splitn(..., 2)` makes it equivalent

Reply:
And you are, of course, correct.  Can I start today again please, this time with more tea...

Reply:
> move to common

Ooo yeah we're probably duplicating a few of these arg parsers.

Reply:
How similar this is to the environment variable parser will shock you.